### PR TITLE
Move to Compose1.1.1 with the release notes

### PR DESCRIPTION
**Pull request by joyl1216**

### CODE REVIEW

File: ComposeTesting/dependencies.gradle [hunk 1]
```diff
@@ -14,11 +14,11 @@ ext {
 
     // ComposeTesting library version code:
     // If you want to publish a new version, bump in one (1) the specific line(s)
-    composeTestingVersionCode = 2
+    composeTestingVersionCode = 3
```
Comment (khalp) [inline ComposeTesting/dependencies.gradle:17]: Do you think we should wait until the ComposeTesting update to bump the version? I'm ok with it either way, just want to see what you think!

**Review by khalp — APPROVED** [review]
Looks great, I think it's a good addition to the README!

In the future as we release new SDK versions, is the plan to update the release notes after the artifacts are uploaded to Maven Central? Or is this something we'll do as part of the PR before the artifacts are released (like how we update the compose version badge)?

File: ComposeTesting/dependencies.gradle (hunk 1 above)
Comment (joyl1216) [reply ComposeTesting/dependencies.gradle:17]: Yes, we should wait. The same as the sample PR.

**Comment by joyl1216** [comment]
> Looks great, I think it's a good addition to the README!
> 
> In the future as we release new SDK versions, is the plan to update the release notes after the artifacts are uploaded to Maven Central? Or is this something we'll do as part of the PR before the artifacts are released (like how we update the compose version badge)?

I think we should update the "latest update" until we push the version to Maven Central and it is available to the public. But it is ok for us to update the version number in README if we are going to release the library soon, such as in a couple of hours.

**Pull request merged**